pandas/io/pytables.py

```diff
@@ -716,11 +712,9 @@ def open(self, mode: str = "a", **kwargs):
                     "which allows\n"
                     "files to be opened multiple times at once\n"
                 )
-
             raise err
 
         except Exception as err:
```

Comment:
if you're up for it, narrowing down what we catch here would be really helpful

Reply:
I did that, looking at the source code of ``tables``. But it turns out that there was (is) no test for this exception in pandas code. So I will need to figure out how to do it, but it will require some mocking, including package version and stuff.

Reply:
Taking into account that pytables version >= 3.4.3 in current installation requirements (https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html), then I can probably remove ``ver >= 3.1`` statement.